xml/System/Environment.xml

```diff
@@ -1639,16 +1617,18 @@ The following example creates environment variables for the <xref:System.Environ
         <ReturnType>System.OperatingSystem</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets an <see cref="T:System.OperatingSystem" /> object that contains the current platform identifier and version number.</summary>
-        <value>An object that contains the platform identifier and version number.</value>
+        <summary>Gets the current platform identifier and version number.</summary>
+        <value>The platform identifier and version number.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks
 
-The `Environment.OSVersion` property does not provide a reliable way to identify the exact operating system and its version. Therefore, we do not recommend that you use this method. Instead: 
+In .NET 5.0 and later versions, the `Environment.OSVersion` property returns the actual version of the operating system. For more information, see [Environment.OSVersion returns the correct operating system version](/dotnet/core/compatibility/corefx#environmentosversion-returns-the-correct-operating-system-version).
+
+In .NET Framework and .NET Core 1.0 - 3.1, this property does *not* provide a reliable way to identify the exact operating system version. Instead of checking this property, consider one of these options:
```

Comment:
I don't think we should steer folks toward RuntimeInformation for version checks.  I would just caveat the behavior of OSVersion (only on Windows) that it can be subject to compatibility shims.   I think its fine to steer folks to RuntimeInformation for OS family checks as those are easier and specifically what RuntimeInformation was designed for.  /cc @jkoritzinsky @stephentoub